tidb-lightning/tidb-lightning-overview.md

```diff
@@ -47,6 +47,7 @@ TiDB Lightning also supports using TiDB-backend for data import. In this mode, `
 
 ## Restrictions
 
-If you use TiDB Lighting together with TiFlash:
+- If you use TiDB Lighting together with TiFlash:
 
-No matter a table has TiFlash replica(s) or not, you can import data to that table using TiDB Lightning. Note that this might slow the TiDB Lightning procedure, which depends on the NIC bandwidth on the lightning host, the CPU and disk load of the TiFlash node, and the number of TiFlash replicas.
+    No matter a table has TiFlash replica(s) or not, you can import data to that table using TiDB Lightning. Note that this might slow the TiDB Lightning procedure, which depends on the NIC bandwidth on the lightning host, the CPU and disk load of the TiFlash node, and the number of TiFlash replicas.
+- For Apache Parquet files, TiDB Lightning currently only accepts Amazon Aurora Parquet files.
```

Comment:
```suggestion
    No matter a table has TiFlash replica(s) or not, you can import data to that table using TiDB Lightning. Note that this might slow the TiDB Lightning procedure, which depends on the NIC bandwidth on the lightning host, the CPU and disk load of the TiFlash node, and the number of TiFlash replicas.

- For Apache Parquet files, TiDB Lightning currently only accepts Amazon Aurora Parquet files.
```